parkbenchcommon/ramdisk.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Joel Allen Luellwitz and Emily Frost
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Provides a class for managing ramdisks."""
```

Comment:
Maybe get rid of the word 'Provides'. Also, there should not be a space after the triple quotes.

Reply:
This is module documentation, distinct from the class documentation. I don't think it's really important either way, but this is technically correct.

Reply:
You win.